src/main/java/org/jfrog/hudson/pipeline/common/executors/GradleExecutor.java

```diff
@@ -69,7 +69,7 @@ public void execute() throws Exception {
         ExtractorUtils.addVcsDetailsToEnv(new FilePath(ws, rootDir), extendedEnv, listener);
         tempDir = ExtractorUtils.createAndGetTempDir(ws);
         EnvExtractor envExtractor = new MavenGradleEnvExtractor(build,
-                buildInfo, deployer, gradleBuild.getResolver(), listener, launcher, tempDir, extendedEnv);
+                buildInfo, deployer, gradleBuild.getResolver(), listener, launcher, tempDir, extendedEnv, this.gradleBuild.isUsesPlugin());
```

Comment:
None of the other variables use "this"
```suggestion
                buildInfo, deployer, gradleBuild.getResolver(), listener, launcher, tempDir, extendedEnv, gradleBuild.isUsesPlugin());
```